plugins/MauticCrmBundle/Integration/SugarcrmIntegration.php

```diff
@@ -1911,4 +1920,35 @@ public function convertSuiteCrmToMauticMultiSelect($suiteCrmMultiSelectStringToC
 
         return $convertedString;
     }
+
+    
```

Comment:
@tsummerer Travis CI is complaining about these 2 rows. There should be only one.

If you want the fixer to fix it for you then run `bin/php-cs-fixer fix -v`. But it may take a while.